core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -889,16 +870,40 @@ class DAGSchedulerSuite extends SparkFunSuite with LocalSparkContext with TimeLi
    * @param stageId - The current stageId
    * @param attemptIdx - The current attempt count
    * @param numShufflePartitions - The number of partitions in the next stage
+   * @param taskSetOpt - The index of task set
+   * @param hostNames - Host on which each task in the task set is executed
+   * @param mapTaskIds - ID of each task in the task set
+   * @param execIds - ID of the executor in which each task in the task set is executed
    */
   private def completeShuffleMapStageSuccessfully(
```

Comment:
Well, there's another concern that while we're reusing the function in more places, it gets more complicated to use because of new arguments.

Reply:
@Ngone51 Thanks! Only two test cases passed maptaskids. I will delete this parameter.

Reply:
Only two test cases passed `execIds `. I will delete this parameter too.